frontend/src/Editor/component-properties-resolution.js

```diff
@@ -27,3 +27,19 @@ export const resolveStyles = (component, currentState, defaultValue, customResol
     return {};
   }
 };
+
+export const resolveGeneral = (component, currentState, defaultValue, customResolvables) => {
+  if (currentState) {
+    const general = component.definition.general;
```

Comment:
Existing widgets seems to be crashing without this.
```suggestion
    const general = component.definition?.general ?? {};
```